tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -2809,6 +2819,27 @@ public function userGetsDashboardWidgets($user, $apiVersion = 'v1', ?TableNode $
 		}
 	}
 
+	/**
+	 * @Then /^user "([^"]*)" sees the following entry when loading the dashboard conversations \((v4)\)$/
+	 *
+	 * @param string $user
+	 * @param string $apiVersion
+	 * @param ?TableNode $formData
+	 */
+	public function userGetsEventConversationsForTalkDashboard(string $user, string $apiVersion, ?TableNode $formData = null): void {
+		$this->setCurrentUser($user);
+		$this->sendRequest('GET', '/apps/spreed/api/' . $apiVersion . '/dashboard/events');
+		$this->assertStatusCode($this->response, 200);
+
+		$data = $this->getDataFromResponse($this->response);
+
+		array_map(static function ($room) {
```

Comment:
missing the actual assert with the data array?
```suggestion
		$expected = array_map(static function ($room) {
```

and afterwards:
```php
$this->assertRooms(); // 90% sure on the name, but the function exists
```